membership_account_balance/__init__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```

Comment:
@elicoidal travis has 2 main issues: The module membership_accont_balance has deprecated syntax, Would I change its code?; Other modules except membership_... has many warning/errors,  How to handle them?